go/vt/throttler/max_replication_lag_module_test.go

```diff
@@ -96,6 +104,47 @@ func (tf *testFixture) checkState(state state, rate int64, lastRateChange time.T
 	return nil
 }
 
+func TestNewMaxReplicationLagModule_recalculateRate(t *testing.T) {
+	testCases := []struct {
+		name        string
+		lagRecord   replicationLagRecord
+		expectPanic bool
+	}{
+		{
+			name: "Zero lag",
```

Comment:
nit: Isn't this zero time, not zero lag?

Reply:
I called it _Zero lag_ because it matches the check [here](https://github.com/vitessio/vitess/pull/12620/files/8b3f65464e7939606e85fabbe05548f1491a3f78#diff-76abba21a59ffa4bcc4edd2a163c98835cc2d56ab62db20e5858781d6a1fe732R302) for `lagRecordNow.isZero()`.